admin/controllers/tech-support.js

```diff
@@ -552,6 +553,41 @@ const controller = {
     } catch (error) {
       return next(error)
     }
+  },
+
+  getPsReportRun: async function getPsReportRun (req, res, next) {
+    try {
+      res.locals.pageTitle = 'PS Report Run'
+      req.breadcrumbs('PS Report Run')
+      res.render('tech-support/ps-report-run', {
+        breadcrumbs: req.breadcrumbs(),
+        runReport: req.body?.runReport ?? false,
+        response: ''
+      })
+    } catch (error) {
+      return next(error)
+    }
+  },
+
+  postPsReportRun: async function postPsReportRun (req, res, next) {
+    const runReport = req.body.runReport === 'true'
+    const response = runReport === true ? 'PS Report Requested' : 'Report NOT Requested - checkbox tick required'
```

Comment:
I don't see why you are mixing types here.  `runReport` is sometimes a string and sometimes a bool.   If we need a boolean flag for the server logic, then I would prefer a separate variable that is always a bool that is derived from the incoming form post.

Reply:
i am creating a boolean and setting its state based upon the string value of the checkbox element.
It does read a bit wordy, so i'll trim it back and just use the form element string value.